server/issue.go

```diff
@@ -107,28 +108,29 @@ func (s *Server) CleanOutdatedIssues() {
 		issues = result.Data.([]*model.Issue)
 	}
 
-	mlog.Info("Will process the Issues", mlog.Int("Issues Count", len(issues)))
+	mlog.Info("Processing Issues", mlog.Int("Issues Count", len(issues)))
 
 	for _, issue := range issues {
-		ghIssue, _, errIssue := s.GithubClient.Issues.Get(context.Background(), issue.RepoOwner, issue.RepoName, issue.Number)
-		if errIssue != nil {
-			mlog.Error("Error getting Pull Request", mlog.String("RepoOwner", issue.RepoOwner), mlog.String("RepoName", issue.RepoName), mlog.Int("PRNumber", issue.Number), mlog.Err(errIssue))
-			if _, ok := errIssue.(*github.RateLimitError); ok {
-				mlog.Error("GitHub rate limit reached")
-				s.CheckLimitRateAndSleep()
-			}
+		ghIssue, _, err := s.GithubClient.Issues.Get(context.Background(), issue.RepoOwner, issue.RepoName, issue.Number)
```

Comment:
From looking at the code it seams like there are no uses of `CleanOutdatedIssues`. Can we just drop it alongside `SqlIssueStore.List()` and `SqlIssueStore.ListOpen()`?